data/translations/nl.ts

```diff
@@ -0,0 +1,417 @@
+        <source>The OpenRazer daemon is not installed</source>
+        <translation>De OpenRazer-achtergronddienst is niet geïnstalleerd</translation>
+    </message>
+    <message>
+        <location filename="../../src/razergenie.cpp" line="66"/>
+        <source>The daemon is not installed or the version installed is too old. Please follow the installation instructions on the website!
+
+If you are running RazerGenie as a flatpak, you will still have to install OpenRazer outside of flatpak from a distribution package.</source>
+        <translation>De achtergronddienst is niet geïnstalleerd of de geïnstalleerde versie is verouderd. Volg de installatie-instructies op de website!
+
+Als je RazerGenie uitvoert als flatpak, dan dien je OpenRazer buiten Flatpak te installeren via de pakketbronnen van je distributie.</translation>
+    </message>
+    <message>
+        <location filename="../../src/razergenie.cpp" line="67"/>
+        <source>Open OpenRazer website</source>
+        <translation>OpenRazer-website openen</translation>
+    </message>
+    <message>
+        <location filename="../../src/razergenie.cpp" line="69"/>
+        <location filename="../../src/razergenie.cpp" line="85"/>
+        <location filename="../../src/razergenie.cpp" line="103"/>
+        <source>Open settings</source>
+        <translation>Instellingen openen</translation>
+    </message>
+    <message>
+        <location filename="../../src/razergenie.cpp" line="83"/>
+        <source>The OpenRazer daemon is not available.</source>
+        <translation>De OpenRazer-achtergronddienst is niet beschikbaar.</translation>
+    </message>
+    <message>
+        <location filename="../../src/razergenie.cpp" line="84"/>
+        <source>The OpenRazer daemon is not started and you are not using systemd as your init system.
+You have to either start the daemon manually every time you log in or set up another method of autostarting the daemon.
+
+Please consult the documentation for details.</source>
+        <translation>De OpenRazer-achtergronddienst is niet actief je maakt geen gebruik van systemd als init-systeem.
+Start de dienst handmatig telkens als je je aanmeldt of zoek een andere manier om deze automatisch op te starten.
+
+Meer informatie hierover is te vinden in de documentatie.</translation>
+    </message>
+    <message>
+        <location filename="../../src/razergenie.cpp" line="98"/>
+        <source>The OpenRazer daemon is currently not available. The status output is below.</source>
+        <translation>De OpenRazer-achtergronddienst is momenteel niet beschikbaar. Hieronder vind je de statusuitvoer.</translation>
+    </message>
+    <message>
+        <location filename="../../src/razergenie.cpp" line="100"/>
+        <source>If you think, there&apos;s a bug, you can report an issue on GitHub:</source>
+        <translation>Als je vermoedt dat dit een bug is, meld dit voorval dan op GitHub:</translation>
```

Comment:
```suggestion
        <translation>Als je vermoedt dat er een bug is, meld het dan op GitHub:</translation>
```

Reply:
"dit" refers to the previous error message, so that's correct and "het" wouldn't be correct 'cause in Dutch it's "de bug", not "het bug".

Reply:
The English message doesn't refer to the previous message though. If it would be "If you think this is a bug", then your translation would be accurate. However, it's talking about a bug in general, not just the previous error message.